Solution1/ConsoleApp/Services/UserCommunicateService.cs

```diff
@@ -27,10 +27,13 @@ public UserCommunicateService(ILogger logger, IInvoker invoker, IPerformerComman
 
         public async Task<bool> CommunicateAsync()
         {
+            var startNumberPoint = 0;
+
             Console.WriteLine("Select menu item:");
-            Console.WriteLine("0 - Exit");
-            Console.WriteLine("1 - Get currently weather");
-            Console.WriteLine("2 - Get weather for a period of time");
+            Console.WriteLine($"{startNumberPoint++} - Exit");
```

Comment:
overhead. in switch you do cases with hardcoded numbers, so no sense to do this war and to make string building more complicated

Reply:
+